pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -116,11 +116,13 @@ function getSpaceAvailable(storagePools, connectionName) {
     return space;
 }
 
-function validateParams(vmParams) {
+function validateParams(vmParams, vms) {
     const validationFailed = {};
 
     if (isEmpty(vmParams.vmName.trim()))
         validationFailed.vmName = _("Name must not be empty");
+    else if (vms.some(vm => vm.name === vmParams.vmName))
+        validationFailed.vmName = _("VM ") + vmParams.vmName + _(" already exists");
```

Comment:
You can use cockpit.format for that,
cockpit.format(_("VM $0 already exists"), vmParams.vmName);

I think it looks nicer tbh. And tranlations won't be weird string finishing/starting with spaces.

Reply:
Moreover, we must not enforce word ordering in translatable strings, other languages may put the parameter at a different position.